gnovm/stdlibs/std/params.go

```diff
@@ -18,6 +18,13 @@ type ParamsInterface interface {
 	SetUint64(key string, val uint64)
 	SetBytes(key string, val []byte)
 	SetStrings(key string, val []string)
+
+	GetString(key string) (string, bool)
+	GetBool(key string) (bool, bool)
+	GetInt64(key string) (int64, bool)
+	GetUint64(key string) (uint64, bool)
+	GetBytes(key string) ([]byte, bool)
+	GetStrings(key string) ([]string, bool)
```

Comment:
We should avoid this approach, as it will create an alternative to the GNO storage that we prefer to use.

If you want to obtain the value, you can store it twice: once in the params module for the chain and once in the contract. You can then read it from the same contract or from other contracts by importing it.

Reply:
Dropped:

[22908ed](https://github.com/gnolang/gno/pull/3999/commits/22908ed255c55ed89e8a8259254c36fb9f38051b)

I can see us needing to fetch params from realms in the future